temporal-spring-boot-autoconfigure/src/main/java/io/temporal/spring/boot/autoconfigure/template/WorkersTemplate.java

```diff
@@ -512,6 +512,48 @@ private void configureWorkflowImplementationAutoDiscovery(
 
   @SuppressWarnings("unchecked")
   private <T> void configureWorkflowImplementation(Worker worker, Class<?> clazz) {
+    // Handle dynamic workflows
+    if (DynamicWorkflow.class.isAssignableFrom(clazz)) {
```

Comment:
I figure lots of this comes from existing code? I'm guessing there's not a reasonable way to reuse (i.e. maybe we don't want to reference `internal` packages of another JAR)?

Reply:
No we don't want to reference internal, it will break Java modules.